src/services/FacilityService.ts

```diff
@@ -0,0 +1,14 @@
+const KEYS = {
+    facilities: "facilities",
+  };
+  export function getAllFacilities() {
+    if (localStorage.getItem(KEYS.facilities) == null)
+      localStorage.setItem(KEYS.facilities, JSON.stringify([]));
+    return myPromise.then((value) => value);
+  }
```

Comment:
export const getAllFacilities = () => {
    const facilityes = localStorage.getItem(KEYS.facilities);
    if (!facilityes) {
        return [];
    }
    return facilityes;
}

Reply:
change it to async/await